### ISSUE TITLE
Beta2  still doesn't compile on Flutter 3

### ISSUE THREAD

**Issue by charafau** [GetDutchie/datadog_flutter#105]
hi! looks like there's stll an error when using Flutter 3

```
e: /Users/rafal.wachol/.pub-cache/hosted/pub.dartlang.org/datadog_flutter-2.0.0-beta.2/android/src/main/kotlin/com/greenbits/datadog_flutter/DatadogFlutterPlugin.kt: (35, 5): Unresolved reference: context
e: /Users/rafal.wachol/.pub-cache/hosted/pub.dartlang.org/datadog_flutter-2.0.0-beta.2/android/src/main/kotlin/com/greenbits/datadog_flutter/DatadogFlutterPlugin.kt: (63, 20): Unresolved reference: context

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':datadog_flutter:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

```

Looks like companion object was removed together with `context` variable:

https://github.com/GetDutchie/datadog_flutter/blob/8f38c649d65e816b57dbf4c56dbd44b0e2d70513/android/src/main/kotlin/com/greenbits/datadog_flutter/DatadogFlutterPlugin.kt#L50

**Comment by rodrigobastosv** [comment]
I'm getting the same error. Hope the pr will be merged and the problem will go away soon

**Comment by tshedor** [comment]
@charafau @rodrigobastosv this is released in `2.0.0-beta.4`

**Comment by charafau** [comment]
thanks for fast merge !
